lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoAnnotationDeclaration.kt

```diff
@@ -8,6 +8,8 @@ class KoAnnotationDeclaration private constructor(
 ) : KoNamedDeclaration(ktAnnotationEntry) {
     override val name by lazy { ktAnnotationEntry.shortName.toString() }
 
+    override val text: String by lazy { ktAnnotationEntry.text.removePrefix("@") }
```

Comment:
test should always contain full test of the declaration, this is mostly used for debugging, so this text should not be changed